### DISCUSSION TITLE
Installation aborted: Unable to untar sbfspot-sqlite-arm64-linux-bullseye.tar.gz

### DISCUSSION THREAD

**Discussion by RuudKapteijn** [SBFspot/sbfspot-config#56] [Q&A] [answered]
Installation report:

rka@SBFspot:~ $ curl -s https://raw.githubusercontent.com/sbfspot/sbfspot-config/master/sbfspot-config | sudo bash
SBFspot Configuration Tool V1.3.3
Running on Raspberry Pi 3 Model B Rev 1.2
Checking whiptail... Installed.
architecture=arm64
Debian GNU/Linux 11 (bullseye)
username=rka
userhome=/home/rka
Minimum SBFspot version supported by this tool is 3.5.0
Request to install latest version (3.9.6)
Searching for existing configuration(s)...
Reading defaults from /var/tmp/sbfspot.3/sbfspot.defaults... Done
Saving defaults to /var/tmp/sbfspot.3/sbfspot.defaults... Done
Performing some checks before installation... Done
Downloading sbfspot-sqlite-arm64-linux-bullseye.tar.gz
tar: sbfspot-sqlite-arm64-linux-bullseye.tar.gz: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
Unable to untar sbfspot-sqlite-arm64-linux-bullseye.tar.gz
Installation aborted

Any suggestions what is going wrong here?

**Comment by groenerik** [comment]
I experience exactly the same:

pi@raspberrypi:~ $ curl -s https://raw.githubusercontent.com/sbfspot/sbfspot-config/master/sbfspot-config|sudo bash
SBFspot Configuration Tool V1.3.3
Running on Raspberry Pi 3 Model B Rev 1.2
Checking whiptail... Installed.
architecture=armhf
Raspbian GNU/Linux 10 (buster)
username=pi
userhome=/home/pi
Minimum SBFspot version supported by this tool is 3.5.0
Request to install latest version (3.9.6)
Searching for existing configuration(s)...
Reading defaults from /var/tmp/sbfspot.3/sbfspot.defaults... Done
Saving defaults to /var/tmp/sbfspot.3/sbfspot.defaults... Done
Performing some checks before installation... Done
Downloading sbfspot-nosql-arm-linux-buster.tar.gz
tar: sbfspot-nosql-arm-linux-buster.tar.gz: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
Unable to untar sbfspot-nosql-arm-linux-buster.tar.gz
Installation aborted
pi@raspberrypi:~ $

**Comment by sbf-** [comment, marked as answer]
[Fixed](https://github.com/SBFspot/sbfspot-config/issues/57)

**Reply by RuudKapteijn** [reply]
Fix confirmed, installation successfull
Many thx, Ruud.

**Comment by groenerik** [comment]
Fix confirmed. Installation complete. Many thanks.
